README.md

```diff
@@ -37,7 +37,7 @@ Runtime Environment Variables:
 | :------------------- | :---------------------------: | -----------------------------------------------------------------------------: |
 | contact_name         |       Abinash Konersman       | name of environment variable contact_name containing name of the cotact person |
```

Comment:
Now that Abinash is out, we should probably remove his name/email so we have more accurate contacts. Can you instead swap in Ashley?

Reply:
changed to Ashley Nylen